nat-lab/tests/config.py

```diff
@@ -18,17 +18,25 @@ def get_root_path(path: str) -> str:
 LINUX_VM_PRIMARY_GATEWAY = "10.55.0.10"
 LINUX_VM_SECONDARY_GATEWAY = "10.66.0.10"
 
+PRIMARY_VM_NETWORK_PREFIX = "10.55.0"
+SECONDARY_VM_NETWORK_PREFIX = "10.66.0"
+
+# https://vagrant-libvirt.github.io/vagrant-libvirt/configuration.html#management-network
+LIBVIRT_MANAGEMENT_NETWORK_PREFIX = "192.168.121"
+
```

Comment:
As I see these consts are used only in one file, maybe it would be more convenient to keep them there?

Reply:
I think the idea for `config.py` is to keep all Vagrant, Docker constants in a single place, to make it easier to make changes without having to change these magic constants throughout the code.